morpheus-apollo-1/genesis.json

```
@@ -30,51 +30,11142 @@
+          ]
+        },
+        {
+          "address": "desmos1hrncsvcx47afflsau2zmcd6lgea9m02l3p4k5s",
+          "coins": [
+            {
+              "denom": "udaric",
+              "amount": "10000000"
+            }
+          ]
+        },
+        {
+          "address": "desmos1ernh92vrx95fm3a3cyy4am6xc5y0fvavux6gze",
+          "coins": [
+            {
+              "denom": "udaric",
+              "amount": "10000000"
+            }
+          ]
+        },
+        {
+          "address": "desmos1ph2af9y36dkvsp5vf8c2q2r9pf292aaf9hqmf6",
+          "coins": [
+            {
+              "denom": "udaric",
+              "amount": "10000000"
+            }
+          ]
+        },
+        {
+          "address": "desmos1e7g9l8kcdky5ajetrsr5huthg02dk2wkyljksz",
+          "coins": [
+            {
+              "denom": "udaric",
+              "amount": "10000000"
+            }
+          ]
+        },
+        {
+          "address": "desmos1aa5atf0mem08fgefhcsu6rc0qd6as2yydh4e4m",
+          "coins": [
+            {
+              "denom": "udaric",
+              "amount": "10000000"
+            }
+          ]
+        },
+        {
+          "address": "desmos1vm8dd23276tsxrye4cdreg52esvuc3rml9devc",
+          "coins": [
+            {
+              "denom": "udaric",
+              "amount": "10000000"
+            }
+          ]
+        },
+        {
+          "address": "desmos1aatv3x3hgh06h72gjyyjtgvva2cm7x3tuvle3n",
+          "coins": [
+            {
+              "denom": "udaric",
+              "amount": "10000000"
+            }
+          ]
+        },
+        {
+          "address": "desmos1huewk6j8tyjy6xjwkcuk6syecfllkty0a0kns4",
+          "coins": [
+            {
+              "denom": "udaric",
+              "amount": "10000000"
+            }
+          ]
+        },
+        {
+          "address": "desmos16gcntemxx2ghvqg0zkyzm8tpy9up8u6guyda5h",
+          "coins": [
+            {
+              "denom": "udaric",
+              "amount": "10000000"
+            }
+          ]
+        },
+        {
+          "address": "desmos1jvqpec94x652s5k6gg22q0ppxpus8yz7qx8v7n",
+          "coins": [
+            {
+              "denom": "udaric",
+              "amount": "10000000"
+            }
+          ]
+        },
+        {
+          "address": "desmos1l7j6w8t282p3jnngz6e096y3wnk8j7nz2y052q",
+          "coins": [
+            {
+              "denom": "udaric",
+              "amount": "10000000"
+            }
+          ]
+        },
+        {
+          "address": "desmos1awrg9y8lv7695as65rjcngl77hzxz5h4w69pap",
+          "coins": [
+            {
+              "denom": "udaric",
+              "amount": "10000000"
+            }
+          ]
+        },
+        {
+          "address": "desmos1ct2uu96e80xq3d6x4ual4d4sxna678p9qxaryt",
+          "coins": [
+            {
+              "denom": "udaric",
+              "amount": "10000000"
+            }
+          ]
+        }
```

Add bank balances for the vesting account
```suggestion
        },
        {
          "address": "desmos1snmetx9sfrmznr4z6d3lnwlyvwu36sgcpzzhlr",
          "coins": [
            {
              "denom": "udaric",
              "amount": "300000000000"
            }
          ]
        }
```